core/platform/email/gae_email_services.py

```diff
@@ -79,14 +80,25 @@ def send_mail(
     if not mail.is_email_valid(recipient_email):
         raise ValueError(
             'Malformed recipient email address: %s' % recipient_email)
-
-    if bcc_admin:
-        mail.send_mail(
-            sender_email, recipient_email, subject, plaintext_body,
-            html=html_body, bcc=[feconf.ADMIN_EMAIL_ADDRESS])
-    else:
-        mail.send_mail(
-            sender_email, recipient_email, subject, plaintext_body,
-            html=html_body)
+    if feconf.EMAIL_SERVICE_PROVIDER == feconf.GAE_EMAIL_SERVICE:
```

Comment:
Instead of putting the choice here, can we put it in core.platform.models._Gae.import_email_services instead, and make sure that gae_email_services.py and mailgun_email_services.py have identical APIs? I think it's cleaner that way.